packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -427,6 +427,14 @@ export function createRococo (t: TFunction): EndpointOption {
           Valibre: 'wss://testnet.valibre.dev'
         }
       },
+      {
+        info: 'rococoLoomNetwork',
+        paraId: 2043,
+        text: t('rpc.rococo.loomnetwork', 'Loom Network', { ns: 'apps-config' }),
```

Comment:
Sort alphabetical - this is the failing test.

Reply:
Ok, fixed, `yarn test` is passing locally now.